mobile.js

```diff
@@ -0,0 +1,28 @@
+const modalOpen = document.getElementById('open-modal');
+const modalContainer = document.getElementById('container-menu-modal');
+const modalClose = document.getElementById('btn-close');
+const blurContainer = document.getElementById('first-container');
+const mainContainer = document.getElementById('main');
+
+const liPortfolio = document.getElementById('li-portfolio');
+const liAbout = document.getElementById('li-about');
+const liContact = document.getElementById('li-contact');
+
+function openModal() {
+  blurContainer.classList.add('blurize');
+  mainContainer.classList.add('blurize');
+  modalContainer.classList.add('turn_on');
+}
+
+function closeModal() {
+  modalContainer.classList.remove('turn_on');
+  mainContainer.classList.remove('blurize');
+  blurContainer.classList.remove('blurize');
+}
+
+modalOpen.addEventListener('click', openModal);
+modalClose.addEventListener('click', closeModal);
+
+liPortfolio.addEventListener('click', closeModal);
+liAbout.addEventListener('click', closeModal);
+liContact.addEventListener('click', closeModal);
```

Comment:
- [ ] You did an excellent job working on this Milestone! 🚀 However, regarding JavaScript best practices, this piece of code feels kind of repetitive, imagine if you want to add a new section to your portfolio in the future, you would have to: first, select the `li` element (As you're selecting them individually by id), and then, add an event listener to close the modal. 🤔 This isn't very practical as you have to repeat yourself a lot, remember the DRY rule from [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)!

To solve this, you can give all these links a common class, something like: `li-element`, then you can select all:

```js
const navigationLinks = document.querySelectorAll('.li-element')
```

This returns a NodeList with all `li-elements` and you can iterate over it and add each one of them the click listener:

```js
navigationLinks.forEach((link) => {
  link.addEventListener('click', closeModal)
}
```

This way, if you ever want to add more links, you just need to give them the `li-element` class and they will have this functionality without you needing to manually set it! Please consider applying this change 🤓 🔧

Reply:
Hi @anagudelogu , I am very grateful for your observation, it helped me to learn more. 👏 

![image](https://user-images.githubusercontent.com/110146573/225210455-b179d973-df82-40c4-9797-3b5d77ce4806.png)